test/integration/routing_test.rb

```diff
@@ -0,0 +1,49 @@
+require "test_helper"
+
+class RoutingTest < ActionDispatch::IntegrationTest
+  def contoller_in_ui?(controller)
+    !controller.nil? && controller !~ /^api|internal.*$/
+  end
+
+  setup do
+    @prev_env = ENV["RAILS_ENV"]
+    ENV["RAILS_ENV"] = "production"
+    routes = Rails.application.routes.routes
+    @ui_paths_verb = routes.map { |r| [r.path.spec.to_s, r.verb] if contoller_in_ui? r.defaults[:controller] }.compact.to_h
+  end
+
+  test "active storate routes don't exist" do
+    assert_raises(ActionController::RoutingError) do
```

Comment:
I tried using shoulda-matcher for this `should_not route('path').to('any#action')`, but got stuck on [this](https://github.com/thoughtbot/shoulda-matchers/issues/1158). Other alternative is rails routing assertions but there is no method for `refute` and using `assert_generates` will need similar `asser_raises`.